### PR TITLE
[#804] Spec: Overhaul (First Tranche)

### PR DESCRIPTION
**Pull request by mkarg**
This PR collects an intial tranche of the spec overhaul organized by issue #804:
- Copyright EF (instead Oracle)
- Jakarta EE Design (incl. 2.x -> 2.2-SNAPSHOT)
- Jakarta EE Similarity
- Java EE Similarity
- Copyright header

**As these are no API changes, this is a fast-track review period of just one day as per our [committer rules](https://github.com/eclipse-ee4j/jaxrs-api/wiki/Committer-Conventions#minimum-length-of-review-period-before-merge--close-of-prs-and-closing-of-issues).**

### CODE REVIEW

**Comment by mkarg** [comment]
@eclipse-ee4j/eclipse-jaxrs Committers, please review this PR ASAP. Thanks. :-)

**Comment by mkarg** [comment]
@eclipse-ee4j/eclipse-jaxrs Committers, please review this PR. It is open since 20 days meanwhile.

**Comment by chkal** [comment]
I know that reviewing this PR is some work. That's why I tried to make it as easy as possible for you by providing a detailed change summary and a diff of the generated PDF. However, I would really appreciate if we could move this one forward.

**Comment by mkarg** [comment]
@eclipse-ee4j/eclipse-jaxrs Committers, please do review this PR ASAP (= "now"). :-)

**Comment by mkarg** [comment]
@spericas @asoldano @kwsutter Review please. :-)

**Comment by chkal** [comment]
@spericas @asoldano @kwsutter @andymc12 The pull request has been created 5(!) weeks ago. Could you please review the changes, so we can proceed with cleaning up the spec document? Or is there anything blocking you from doing so?

File: jaxrs-spec/src/main/asciidoc/chapters/appendix/_change-log.adoc [hunk 1]
```diff
@@ -1,7 +1,19 @@
...
+* of the Eclipse Foundation Specification License v1.0, which is
+* available at https://www.eclipse.org/legal/efsl.php.
+*******************************************************************
+////
+
 [appendix]
 [[change-log]]
 == Change Log
 
+include::_changes-since-2.2-snapshot.adoc[]
```
[the remark this answers is not in the document]
Comment (mkarg) [reply jaxrs-spec/src/main/asciidoc/chapters/appendix/_change-log.adoc]: This should be done in a _subsequent_ PR *after* there had been an majority voting +1 for your new roadmap proposal.

File: jaxrs-spec/src/main/asciidoc/chapters/license/_license-efsl.adoc [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+prepare and distribute derivative works and portions of this document
+in software that implements the specification, in supporting materials
+accompanying such software, and in documentation of such software,
+PROVIDED that all such works include the notice below. HOWEVER, the
+publication of derivative works of this document for use as a technical
+specification is expressly prohibited.
+
+The notice is:
+
+"Copyright (c) 2018 Eclipse Foundation. This software or
```
[the remark this answers is not in the document]
Comment (mkarg) [reply jaxrs-spec/src/main/asciidoc/chapters/license/_license-efsl.adoc]: No, as this is not the copyright line of this file, but the official template of the EFSL which we must not change.

**Comment by mkarg** [comment]
@andymc12 Thank you for your review. This is much appreciated. Sad to see that the majority of committers did not take the time to review this urgently needed PR, which was openend on October 27. :-(

**Pull request merged**